data_prep/project_src.py

```diff
@@ -312,9 +312,8 @@ def _identify_fuzz_targets(out: str, interesting_filenames: list[str],
         # TODO(dongge): Figure out why the path does not match Bazel projects.
         if os.path.basename(short_path) in interesting_filenames:
           interesting_filepaths.append(path)
-        # This should also include .cpp and .cc but exclude headers which
-        # usually don't contain fuzzer definitions.
-        if '.c' in path:
+
+        if path.endswith('.c') or path.endswith('.cpp') or path.endswith('.cc'):
```

Comment:
How about re-using this instead of enumerating the extensions?
https://github.com/google/oss-fuzz-gen/blob/d220e842b36bb6131950bc79242f0407bbc6338d/data_prep/project_src.py#L30

Reply:
done, thanks!